src/Termonad/Lenses.hs

```diff
@@ -47,6 +58,27 @@ $(makeLensesFor
     ''TMNotebook
  )
 
+-- TODO: upstream this to the focuslist package
```

Comment:
Yeah, I completely agree.

Reply:
I made this change in https://github.com/cdepillabout/focuslist/pull/13 and released in https://hackage.haskell.org/package/focuslist-0.1.1.0, but since focuslist is in the stackage LTS, we'll have to keep using `traversalFLItem` defined here for now, and switch to use `traversalFocusItem` from `focuslist-0.1.1.0` when it is hopefully. available in LTS-19.

```suggestion
-- TODO: This should be available in focuslist-0.1.1.0 as traversalFocusItem.
-- focuslist-0.1.1.0 should likely be available in LTS-19.
-- We should delete this function and use traversalFocusItem when Termonad moves to using LTS-19.
```